src/services/sessions.service.ts

```diff
@@ -85,35 +83,7 @@ export class SessionsService {
 	static async getSessionInfo(sessionId: string): Promise<ServiceResponse<ViewerSession>> {
 		try {
 			const { session } = await sessionsClient.get({ sessionId, jsonValues: true });
-
-			if (!session?.eventId) {
-				const sessionConverted = convertSessionProtoToViewerModel(session!);
-
-				return {
-					data: sessionConverted,
-					error: undefined,
-				};
-			}
-
-			const { data: event, error } = await EventsService.getEnriched(session.eventId);
-
-			if (error) {
-				const errorMessage = i18n.t("sessionMissingEventInfoExtended", {
-					error: (error as Error).message,
-					sessionId,
-					eventId: session.eventId,
-					ns: "services",
-				});
-				LoggerService.error(namespaces.sessionsService, errorMessage);
-
-				return { data: undefined, error };
-			}
-
-			const sessionConverted = convertSessionProtoToViewerModel(
-				session!,
-				event?.sourceType,
-				event?.destinationName
-			);
+			const sessionConverted = convertSessionProtoToViewerModel(session!);
```

Comment:
what if the session returned 'undefined'?

Reply:
Refactored a little bit